WcaOnRails/app/controllers/committee_positions_controller.rb

```diff
@@ -0,0 +1,72 @@
+class CommitteePositionsController < ApplicationController
+  before_action :authenticate_user!, except: [ :index, :show ]
+  before_action -> { redirect_unless_user(:can_manage_committees?) }
+
+  def index
+    @committee = Committee.find_by_slug(params[:committee_id])
+    @committee_positions = @committee.committee_positions
+  end
+
+  def new
+    @committee = Committee.find_by_slug(params[:committee_id])
+    @committee_position = @committee.committee_positions.new
+  end
+
+  def edit
+    @committee = Committee.find_by_slug(params[:committee_id])
+    @committee_position = committee_position_from_params_using_slug(@committee.id)
+  end
+
+  def create
+    @committee = Committee.find(params[:committee_id])
+    @committee_position = @committee.committee_positions.new(committee_position_params)
+    if @committee_position.save
+      flash[:success] = "Created new committee position:" + @committee_position.name
+      redirect_to committee_committee_positions_path(@committee.slug)
+    else
+      flash[:danger] = "Failed to create committee position:" + @committee_position.name
+      render :new
+    end
+  end
+
+  def update
+    @committee = Committee.find(params[:committee_id])
+    @committee_position = committee_position_from_params()
+    if @committee_position.update_attributes(committee_position_params)
+      flash[:success] = "Updated committee position"
+      redirect_to committee_committee_positions_path(@committee.slug)
+    else
+      flash.now[:danger] = "Failed to update committee position"
+      render :edit
+    end
+  end
+
+  def destroy
+    @committee = Committee.find(params[:committee_id])
+    @committee_position = committee_position_from_params()
```

Comment:
Actually there are a lot of such places, but I think rubocop will inform you about the. Do you run it locally?